### ISSUE TITLE
How to install on Plone5?

### ISSUE THREAD

**Issue by krissik** [plone/plone.app.caching#38]
I have Plone (5.0.7 and 5.1b3) installed. If I go to `http://localhost:8080/Plone/prefs_install_products_form` I can not find plone.app.caching to install it. But it is show in old `portal_quickinstaller`. I belive it is because it is in "not installable" list [here](https://github.com/plone/Products.CMFPlone/blob/master/Products/CMFPlone/factory.py)

How do I install plone.app.caching on Plone 5?

**Comment by gforcada** [comment]
When you create a Plone instance using the advanced dialog you have the option there. If you want to install it with a policy package, the usual metadata.xml dependency is probably enough.

But otherwise you are right, as far as I can see, the only way is through the advanced dialog.

**Comment by jensens** [comment]
Works in 5.2.2.

**Comment by mauritsvanrees** [comment]
Wait, let me try. First Plone 5.2.1 because I wonder how it was there. Start with only the Products.CMFPlone egg:

```
[buildout]
extends = https://dist.plone.org/release/5.2.1/versions.cfg
parts = instance

[instance]
recipe = plone.recipe.zope2instance
eggs = Products.CMFPlone
user = admin:admin
```

After running buildout:

```
$ grep caching bin/instance 
  '/Users/maurits/shared-eggs/cp27m/z3c.caching-2.2-py2.7.egg',
  '/Users/maurits/shared-eggs/cp27m/plone.caching-1.2.1-py2.7.egg',
```

Create a Plone Site. Logically, the caching control panel is not there, nor is caching in the Add-ons control panel, because `plone.app.caching` is no dependency. Fine.

Now set `Plone` in the eggs.  This *does* depend on `plone.app.caching`. Run buildout, start Plone. Caching is still nowhere to be seen in the existing site. Not so good. Create a fresh Plone site and caching is there.

I tried on Plone 5.2.1, but it is the same there, nothing changed.

In other words: if you start out with only `Products.CMFPlone`, and only later the buildout to use the `Plone` egg, or explicitly add `plone.app.caching`, then you have no way in the Plone UI to install it.
Not very nice, but I guess this does not happen often.

And I guess in the Plone versions from the first comment, there really was a problem also when you started with the full `Plone` egg.
